assets/src/blocks/Spreadsheet/SpreadsheetEditorScript.js

```diff
@@ -22,9 +22,13 @@ const colors = [
 ];
 
 if (isNewIdentity) {
+  const newColorValues = ['#167f82', '#198700', '#45494c'];
```

Comment:
Maybe here it would be simpler if we just override the `colors`? A bit longer to write maybe, but I think it would make it easier to understand